### PR TITLE
[FEAT] Add stock schema 

### PR DESCRIPTION
**Pull request by YungiKim1999**
Adding the stock schema to the backend of the application

### CODE REVIEW

File: backend/src/mongodb/schemas/stockSchema.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
```
Comment (afei088) [inline backend/src/mongodb/schemas/stockSchema.js:3]: What is the point of this? I don’t think schema is being used?

Comment (YungiKim1999) [reply backend/src/mongodb/schemas/stockSchema.js:3]: i use it for the _id object where I store the user id. I specify that the object is like a mongodb objectId type.
I based this off of this:
https://mongoosejs.com/docs/populate.html

Comment (afei088) [reply backend/src/mongodb/schemas/stockSchema.js:3]: I think you can use Schema({ stockName: {},....}) instead then. Not sure if that will work though?

Comment (YungiKim1999) [reply backend/src/mongodb/schemas/stockSchema.js:3]: that works, new commit changes that

File: backend/src/mongodb/schemas/stockSchema.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+const stockSchema = mongoose.Schema({
```
Comment (afei088) [inline backend/src/mongodb/schemas/stockSchema.js:5]: This is still mongoose.Schema i believe?

**Review by afei088 — APPROVED** [review]
LGTM!

**Pull request merged**